runtime/permissions/prompter.rs

```diff
@@ -38,11 +38,11 @@ pub fn permission_prompt(
 }
 
 pub fn set_prompt_callbacks(
-  before_callback: Option<PromptCallback>,
-  after_callback: Option<PromptCallback>,
+  before_callback: PromptCallback,
+  after_callback: PromptCallback,
```

Comment:
I guess there won't be a situation where we disable this callbacks so I guess it's fine to change the signature here

Reply:
Yeah, we can add it back if we need it.